plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -301,6 +307,47 @@ const config: ControlPanelConfig = {
           },
         ],
         // eslint-disable-next-line react/jsx-key
+        [<h1 className="section-header">{t('X Axis')}</h1>],
+        ['x_axis_format'],
+        [
+          {
+            name: 'x_axis_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Date format'),
+              renderTrigger: true,
+              choices: D3_TIME_FORMAT_OPTIONS,
+              default: 'smart_date',
+              description: D3_FORMAT_DOCS,
+            },
+          },
```

Comment:
Maybe you can create this as another shared control `x_axis_time_format`.

Reply:
My original suggestion was to move this config to `shared_controls` and refer it with the shortcut name.